SpriteTest.java

```diff
@@ -417,6 +417,53 @@ public void testGetSoundBlocksForSpriteEmpty()
         int actual = spritesAnimateTheCrab[1].getSoundBlocksForSprite();
         assertEquals("should be equal", expected, actual);
     }
+
+    /**
+     * Test populateVariables method.
+     */
+    @Test
+    public void testPopulateVariables()
+    {
+        String[] expected = {"sprite1Variable",
+                             "sprite1Variableb"};
+        String[] actual = spritesBigProject[0].getVariables();
+        assertEquals("should be equal", expected, actual);
+    }
+
+    /**
+     * Test populateVariables - empty.
+     */
+    @Test
+    public void testPopulateVariablesEmpty()
+    {
+        String[] expected = new String[0];
+        String[] actual = spritesAnimateTheCrab[1].getVariables();
+        assertEquals("should be equal", expected, actual);
```

Comment:
Use assertArrayEquals here to compare array equality and avoid the deprecation compile warning.